src/esssans/types.py

```diff
@@ -69,6 +76,11 @@
 BeamCenter = NewType('BeamCenter', sc.Variable)
 """Beam center, may be set directly or computed using beam-center finder"""
 
+LowCountThreshold = NewType('LowCountThreshold', sc.Variable)
+"""Threshold below which detector pixels should be masked
+(low-counts on the edges of the detector panel, and the beam stop)"""
```

Comment:
This mask seemed specific to Sans2d, what was the motivation for adding it in a generic place?

Reply:
This needs to be set as a parameter in the notebook.
I think it would be annoying for the user to have to know that for this specific type, they have to do
```Py
params[sans.sans2d.LowCountThreshold] = sc.scalar(100, unit='counts')
```
but just e.g. `SampleRun` for all the other types.

So I am thinking of moving all the sans2d stuff into a folder, and making a `types.py` file in there, where we can then do
```Py
from esssans.types import *
from esssans.sans2d.types import *

params[LowCountThreshold] = sc.scalar(100, unit='counts')
```

I think after adding the loki stuff for the direct beam I wanted to move instrument specifics into subfolders anyway. Does that sound reasonable?

Reply:
I think scoping things is good, not bad. If a param is not generic for a technique then it would be confusing to have it on the "global" level. `import *` may be a antipattern, even here.

I would leave it for now, we can think about this more when adding support for actual ESS instruments.

Reply:
Isn't `sans.sans2d.LowCountThreshold` easier to remember than `sans.sans2d.types.LowCounthreshold`?

I am not at all convinced that having everything in `types` is ideal. Maybe all common params should be on the top level (technique or instrument), details in their submodule (example: reflectometry supermirror calibration with corresponding set of params).